pkg/service/auth_context_test.go

```diff
@@ -256,3 +277,19 @@ func TestEvaluateAnyAuthConfigsWithoutError(t *testing.T) {
 	assert.Check(t, swap)
 	assert.NilError(t, err)
 }
+
+func TestToData(t *testing.T) {
+	var identityConfigs []common.AuthConfigEvaluator
+	identityConfigs = append(identityConfigs, &successConfig{})
+	request := envoy_auth.CheckRequest{}
+	_ = json.Unmarshal([]byte(rawRequest), &request)
+	authContext := newAuthContext(identityConfigs, &request)
+	data := authContext.ToData()
+	if dataJSON, err := json.Marshal(&data); err != nil {
+		t.Error(err)
+	} else {
+		requestJSON, _ := json.Marshal(request.GetAttributes())
+		expectedJSON := fmt.Sprintf(`{"context":%s,"auth":{"identity":null,"metadata":{}}}`, requestJSON)
```

Comment:
So it seems it's AuthN data, or an extended Identity Data (Identity + Metadata)

Reply:
It is the data to perform the authorization. In other words, the strict input that each authorization policy can count on to evaluate its rules.